src/test/java/io/xstefank/wildfly/bot/PRFormatOverrideProjectKeyTest.java

```diff
@@ -0,0 +1,73 @@
+package io.xstefank.wildfly.bot;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTest;
+import io.quarkus.test.junit.QuarkusTest;
+import io.xstefank.wildfly.bot.utils.Action;
+import io.xstefank.wildfly.bot.utils.GitHubJson;
+import io.xstefank.wildfly.bot.utils.Util;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHEvent;
+import org.kohsuke.github.GHRepository;
+
+import java.io.IOException;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.given;
+import static io.xstefank.wildfly.bot.utils.TestConstants.PR_JSON;
+import static io.xstefank.wildfly.bot.utils.TestConstants.TEST_REPO;
+
+/**
+ * Tests for the Wildfly -> ProjectKey function.
+ */
+@QuarkusTest
+@GitHubAppTest
+public class PRFormatOverrideProjectKeyTest {
+
+    private static String wildflyConfigFile;
+    private static GitHubJson gitHubJson;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
we can remove this method completely and just assign in on declaration as it is only one variable? Additionally we can make if `final`